bin/run-wer-automation.sh

```diff
@@ -17,8 +17,8 @@ export ds_dev_batch_size
 ds_test_batch_size=32
 export ds_test_batch_size
 
-ds_training_iters=100
-export ds_training_iters
+ds_epochs=100
+export ds_epochs
 
 ds_validation_step=200
```

Comment:
Should remove this as it has the same effect as `ds_validation_step=0` which is the default.